rts/Lua/LuaOpenGL.cpp

```diff
@@ -3177,8 +3177,8 @@ int LuaOpenGL::PushPopMatrix(lua_State* L)
 	if (arg == 1) {
 		GL::PushMatrix();
 	} else {
-		for (int i = 0; i < (int)matModes.size(); i++) {
-			GL::MatrixMode(matModes[i]);
+		for (const auto matMode: matModes) {
```

Comment:
when the actual type (here `GLenum`) isn't an overly long std::monstrosity, I'd prefer using that over `auto`.